frontends/qt/main.cpp

```diff
@@ -106,6 +106,7 @@ int main(int argc, char *argv[])
     a.setApplicationName(APPNAME);
     a.setOrganizationDomain("shiftcrypto.ch");
     a.setOrganizationName("Shift Cryptosecurity");
+    a.setWindowIcon(QIcon(QCoreApplication::applicationDirPath() + "/bitbox.png"));
```

Comment:
will have to test if this is necessary on Windows/macOS. In any case, like this it might be a problem on those platforms because this file does not exist (you'd need to copy the file in the Makefile also for macOS and windows).

Reply:
Yeah I was afraid about that. Tried to compile it on windows, but, oh man, I will have to rewrite the whole build system before I can successfully do that :P. Could I bundle it into the assets file?

Reply:
Well if you have all the deps installed on windows, `make qt-windows` should work (see AppVeyor ;)). But yeah, Windows is the most painful to build.

assets file: that will probably work and is a good idea!